packages/client/src/pages/Profile/elements/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC, useEffect, useState } from 'react'
+import styles from './profileInfo.module.css'
+import Title from '@components/Title'
+import Avatar from '@pages/Profile/elements/Avatar'
+import Form from '@components/Form'
+import { fieldsConfig } from '@utils/validation/fieldsConfig'
+import { omit } from 'lodash'
+import { useCheckAuthentication } from '@utils'
+import { UserDTO } from '@api/auth/types'
+import { resourcesApi } from '@api'
+import { updateProfile } from '@services/profile'
+import { ErrorResponse } from '@types'
+
+export const ProfileInfo: FC = () => {
+  const [editMode, setEditMode] = useState(false)
```

Comment:
[Предложение]
Обычно для булевых значение используется `is` или `has`. 
Тут по названию непонятно, что значение будет булево. Может добавить `is`?
```suggestion
  const [isEditMode, setIsEditMode] = useState(false)
```